lib/core/auth.js

```diff
@@ -369,7 +370,7 @@ export default class Auth {
 
```

Comment:
I think that the "from" variable will be the full path, containing the base path if fullPathRedirect is enabled. In this case the "isSameURL" call on line 367 will fail, causing an extra, if not infinite redirects.

Reply:
Same with line 348

Reply:
But it is a good practice to use a consistent, instance property for this purpose isn't it?

 If we do that, we can use that variable wherever we want . Also, issue like you [reported](https://github.com/nuxt-community/auth-module/pull/523/files#diff-9a2f0794afc86382d1c3c12125bd13f4R373) can be easily solved using single , predictable fix. In this way, we get better predictability , decoupling/isolation for this module